web/src/hooks/usePasskeyConnection.ts

```diff
@@ -1,35 +1,46 @@
 'use client';
 
 import { logEventSimple } from '@/utils/gtag';
+import { usePasskeyAccount } from '@/providers/PasskeyProvider';
 import storage from 'local-storage-fallback';
-import { useEffect, useState } from 'react';
-import { useConnect } from 'wagmi';
+import { useEffect, useState, useCallback } from 'react';
 
 export default function usePasskeyConnection() {
-  const { connectors, connect, isPending } = useConnect();
   const [initializing, setInitializing] = useState(true);
```

Comment:
discussions: do we need this state? Wouldn't this be the same value as isInitializing?